actions/sign-macos/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:20.04
+USER root
+
+WORKDIR /macnotary/work
+ARG MACNOTARY_BIN=/macnotary/bin
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN mkdir -p /macnotary/work && mkdir -p /macnotary/bin
+RUN apt-get update && apt-get -y install curl unzip
+RUN curl -LO https://macos-notary-1628249594.s3.amazonaws.com/releases/client/latest/linux_amd64.zip && \
```

Comment:
Is this supposed to be public information or should the URL be placed in a secret?

Reply:
I got confirmation that those URLs can be public.